### PR TITLE
Add to render_controlls subroutine

### PR DESCRIPTION
**Pull request by cyblue9**
I created the "render_controlls" subroutine to improve readability.

Please review!

### CODE REVIEW

File: src/display.asm [hunk 1]
```diff
@@ -86,3 +86,17 @@ print_string:
 .done:
   pop ax
   ret
+
+render_controlls:
+  push dx
+  mov dx, 0x0000
+  mov si, left_string
```
Comment (flxbe) [inline src/display.asm:96]: My current calling convention is to push and pop all registers that are manipulated by the callee. This function should therefore also push and pop the `si` register.

**Review by flxbe — CHANGES_REQUESTED** [review]
Minor remark, all in all a nice refactoring.

File: src/display.asm (hunk 1 above)
Comment (flxbe) [inline src/display.asm:96]: The current calling convention is to push and pop all registers that are manipulated by the callee. This should therefore also happen to the `si` register.

**Review by flxbe — CHANGES_REQUESTED** [review]
minor remark, all in all a nice clean up.

File: src/display.asm (hunk 1 above)
Comment (cyblue9) [reply src/display.asm:96]: Oh, I see.
I will fix it soon.

Therefore, I will fix print_window subroutine in the same way.
Is a3727b6 OK?

**Comment by cyblue9** [comment]
Please Re-review!

(Please note a3727b6 commit)

**Review by flxbe — APPROVED** [review]

**Comment by flxbe** [comment]
Nice catch in `print_window`!

**Pull request merged**